WooCommerce/Classes/ViewRelated/Orders/Order Details/Product List Section/Product Details/PickListTableViewCell.swift

```diff
@@ -92,7 +92,12 @@ extension PickListTableViewCell {
                                                        with: item.imageURL?.absoluteString,
                                                        placeholder: UIImage.productPlaceholderImage.imageWithTintColor(UIColor.listIcon),
                                                        progressBlock: nil,
-                                                       completion: nil)
+                                                       completion: { [weak productImageView] (image, error) in
+                                                           guard image != nil else {
+                                                               return
+                                                           }
+                                                           productImageView?.contentMode = .scaleAspectFill
+                                                       })
```

Comment:
Do you think we need this together with similar call in `setupImageView()` below?

Reply:
Ah! Good catch -- I totally missed that!
Fixed in 066b238 😄